app/service-catalog/integrations/datadog.md

```diff
@@ -13,6 +13,33 @@ related_resources:
     url: /gateway/security/
 ---
 
-@todo
+The Datadog integration lets you connect Datadog entities directly to your Service Catalog services.
+
+## Authenticate the Datadog integration
+
+1. From the **Service Catalog** in {{site.konnect_short_name}}, select **[Integrations](https://cloud.konghq.com/service-catalog/integrations)**. 
+1. Select **Datadog**, then **Install Datadog**.
+1. Select your Datadog region and enter your [Datadog API and application keys](https://docs.datadoghq.com/account_management/api-app-keys/). 
+1. Select **Authorize**. 
+
+## Resources
+
```

Comment:
If I remember correctly, a paragraph after a heading/before a table is recommended to improve search

Reply:
done